exir/sym_util.py

```diff
@@ -73,14 +77,14 @@ def eval_shape(shape):
     return new_shape
 
 
-def eval_shape_upper_bound(shape) -> List[int]:
+def eval_shape_upper_bound(shape) -> List[int]:  # pyre-ignore[2]
     new_shape = []
     for _, s in enumerate(shape):
         new_shape.append(eval_upper_bound(s))
     return new_shape
 
 
-def collect_free_symbols(shape) -> Set[sympy.Symbol]:
+def collect_free_symbols(shape) -> Set[sympy.Symbol]:  # pyre-ignore[2]
```

Comment:
can you annotate shape with Union[int, torch.SymInt]?

Reply:
Is this true for all the functions?